src/resources/views/crud/columns/checklist_dependency.blade.php

```diff
@@ -0,0 +1 @@
+@include('crud::columns.checklist')
```

Comment:
Maybe we can do something like this? This will DEFINITELY not work, just an idea 😀

```suggestion
@include('crud::columns.checklist', ['column' => $column['primary']])
@include('crud::columns.checklist', ['column' => $column['secondary']])
```